optuna/visualization/matplotlib/_rank.py

```diff
@@ -0,0 +1,154 @@
+from typing import Callable
+from typing import List
+from typing import Optional
+
+from optuna._experimental import experimental_func
+from optuna.logging import get_logger
+from optuna.study import Study
+from optuna.trial import FrozenTrial
+from optuna.visualization._rank import _get_rank_info
+from optuna.visualization._rank import _get_tick_info
+from optuna.visualization._rank import _RankPlotInfo
+from optuna.visualization._rank import _RankSubplotInfo
+from optuna.visualization.matplotlib._matplotlib_imports import _imports
+
+
+if _imports.is_successful():
+    from optuna.visualization.matplotlib._matplotlib_imports import Axes
+    from optuna.visualization.matplotlib._matplotlib_imports import PathCollection
+    from optuna.visualization.matplotlib._matplotlib_imports import plt
+
+
+_logger = get_logger(__name__)
+
+
+@experimental_func("3.2.0")
+def plot_rank(
+    study: Study,
+    params: Optional[List[str]] = None,
+    *,
+    target: Optional[Callable[[FrozenTrial], float]] = None,
+    target_name: str = "Objective Value",
+) -> "Axes":
+    """Plot parameter relations as scatter plots with colors indicating ranks of objective value.
+
+    Note that, if a parameter contains missing values, a trial with missing values is not plotted.
```

Comment:
Thank you for your suggestion. It looks good change, but I think to apply this change in the follow-up PR because this change should also apply original plotly-version.

Reply:
"values" usually mean objective values, so maybe it's better to say "trials missing the specified parameters will not be plotted."